auth.rb

```diff
@@ -0,0 +1,25 @@
+class Auth
+
+  def initialize(appl)
+    @appl = appl
+  end
+
+  def call(env)
+    begin
```

Comment:
So if I find you in boot camp and ask how it works you will answer, right? :smiling_imp:
[I hope u understand that mentors know how to use google](https://auth0.com/blog/ruby-authentication-secure-rack-apps-with-jwt/)